src/transactionhistory/src/main/java/anthos/samples/bankofanthos/transactionhistory/LedgerReader.java

```diff
@@ -71,9 +72,11 @@ public void startWithCallback(LedgerReaderCallback callback)
             throw new IllegalStateException("callback is null");
         }
         this.callback = callback;
+        this.latestId = STARTING_TRANSACTION_ID;
```

Comment:
Looks like this change didn't make it over to the other LedgerReader. latestId was left uninitialized there

Reply:
Good catch.  Done.